docs/guides/development.md

```diff
@@ -6,6 +6,22 @@
 
 The schema documentation in [`gencode/docs`](../../gencode/docs) must be kept up to date with any changes to the schema. Any additions to the schema should also include [annotations](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.9.1) to describe the new elements. The updated documentation is generated by running the script `bin/gendocs` and its output committed to the repository with the changes. Required python dependencies are in [`etc/requirements.txt`](../../etc/requirements.txt) which must be installed for the script to run.
 
+### Troubleshooting
+
+If these tests are failing, check you are using a compatible system (Linux with
```

Comment:
NB: Prefer not 1st person language for docs.  E.g., "If these tests are failing, check the basic system requirements (Linux with...) --  Just a style thing trying to avoid the tutorial mode of writing.  Or like below ("If..., check...") is ok (just avoid you/I/we is all :-).